packages/next-swc/crates/next-core/src/next_font/google/mod.rs

```diff
@@ -67,7 +67,11 @@ impl ImportMappingReplacement for NextFontGoogleReplacer {
     }
 
     #[turbo_tasks::function]
-    async fn result(&self, request: RequestVc) -> Result<ImportMapResultVc> {
+    async fn result(
+        &self,
+        _context: FileSystemPathVc,
+        request: RequestVc,
+    ) -> Result<ImportMapResultVc> {
```

Comment:
Did @wbinnssmith not land these changes yet?

Reply:
nope